libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -511,18 +528,44 @@ class MapboxNavigation(
          * Use [NavigationOptions.toBuilder] to easily customize selected options.
          */
         @JvmStatic
-        fun defaultNavigationOptions(context: Context): NavigationOptions {
-            return NavigationOptions.Builder(
-                NONE_SPECIFIED,
-                ROUNDING_INCREMENT_FIFTY,
-                DEFAULT_NAVIGATOR_POLLING_DELAY,
-                MapboxDistanceFormatter(
-                    context.applicationContext,
-                    null,
-                    UNDEFINED,
-                    ROUNDING_INCREMENT_FIFTY
+        fun defaultNavigationOptions(context: Context, accessToken: String?): NavigationOptions {
+            val tilesUri = URI("https://api-routing-tiles-staging.tilestream.net")
+            val tilesVersion = "2020_02_02-03_00_00"
+            return NavigationOptions.Builder()
+                .timeFormatType(NONE_SPECIFIED)
+                .roundingIncrement(ROUNDING_INCREMENT_FIFTY)
+                .navigatorPollingDelay(DEFAULT_NAVIGATOR_POLLING_DELAY)
+                .distanceFormatter(
+                    MapboxDistanceFormatter(
+                        context.applicationContext,
+                        null,
+                        UNDEFINED,
+                        ROUNDING_INCREMENT_FIFTY
+                    )
                 )
-            ).build()
+                .onboardRouterConfig(
+                    MapboxOnboardRouterConfig(
+                        obtainOfflineTilesDir(context, tilesUri.host, tilesVersion),
+                        null,
+                        null,
+                        2,
+                        Endpoint(
+                            tilesUri.toString(),
+                            tilesVersion,
+                            accessToken ?: "",
+                            "MapboxNavigationNative"
+                        )
+                    )
+                ).build()
+        }
+
+        private fun obtainOfflineTilesDir(context: Context, host: String, version: String): String {
+            val offlineDir = File(context.filesDir, "Offline/$host/$version")
+            if (!offlineDir.exists()) {
```

Comment:
I believe this is not needed as `offlineDir` is created above either way (even if it doesn't exist).